bitnami/ghost/templates/deployment.yaml

```diff
@@ -204,13 +204,11 @@ spec:
             - name: GHOST_SMTP_USER
               value: {{ .Values.smtpUser | quote }}
             {{- end }}
-            {{- if .Values.smtpPassword }}
```

Comment:
Hi, the pipelines are failing because you need to keep this. If not you are deploying a secret without the key smtp-password